src/pages/court-cases/[courtCaseId]/index.tsx

```diff
@@ -38,16 +39,26 @@ export const getServerSideProps = withMultipleServerSideProps(
     }
     const dataSource = await getDataSource()
 
-    let lockResult: UpdateResult | Error | undefined
+    let courtCase = await getCourtCaseByOrganisationUnit(dataSource, +courtCaseId, currentUser)
```

Comment:
Pre-update fetch from the database to ensure that the court case exists and user has access to the case.